### PR TITLE
Update package-lock to fix react build

### PR DESCRIPTION
**Pull request by hectormmg**

### CODE REVIEW

**Review by konstantin-msft — APPROVED** [review]

File: lib/msal-react/package.json [hunk 1]
```diff
@@ -69,5 +69,8 @@
     "ts-jest": "^29.1.0",
     "tslib": "^2.0.0",
     "typescript": "^4.9.5"
+  },
+  "optionalDependencies": {
+    "@rollup/rollup-linux-x64-gnu": "^4.9.5"
```
Comment (tnorling) [inline lib/msal-react/package.json]: Interesting...why is this just now becoming an issue? Did the VM Image change to something unsupported?

optionalDependencies will be attempted to be installed by customers, we should make this a devDependency or install it ad-hoc in the pipeline itself instead

Comment (hectormmg) [reply lib/msal-react/package.json]: There's an npm bug, I'm still debugging it though

Comment (hectormmg) [reply lib/msal-react/package.json]: This isn't final, just had to open the PR to run the checks

Comment (hectormmg) [reply lib/msal-react/package.json]: https://github.com/npm/cli/issues/4828

**Pull request closed without merging**